content/en/logs/workspaces/use_cases/analyze_inventory_across_regions.md

```diff
@@ -0,0 +1,117 @@
+
+{{< img src="/logs/workspace/use_cases/analyze_inventory_across_regions/retail_region_data_source.png" alt="Data source configuration for store location data with regional information" caption="Data source cell for store location metadata, providing regional context for inventory and sales analysis." style="width:100%;" >}}
+
+## SQL analysis with Log Workspaces
+
+After configuring your data sources, create Analysis cells to run SQL queries that combine and analyze your data. For more information about creating and using Analysis cells with SQL, see [Log Workspaces - Analysis cell][4].
+
+### 1. Identifying products with low stock and high sales
+
+Create an Analysis cell with the following SQL query to join inventory and sales logs, identifying products that need immediate attention:
+
+```sql
+SELECT
+ inventory_product_id,
+ inventory_store_id,
+ inventory_stock_level,
+ inventory_threshold_level,
+ sales_quantity_sold
+FROM
+ inventory_data AS inventory
+JOIN
+ sales_data AS sales
+ ON inventory_product_id = sales_product_id
+ AND inventory_store_id = sales_store_id
+WHERE
+ inventory_stock_level < inventory_threshold_level
+ AND sales_quantity_sold > 30;
+```
+
+{{< img src="/logs/workspace/use_cases/analyze_inventory_across_regions/retail_analysis_cell.png" alt="SQL analysis results showing products with low stock levels and high sales quantities" caption="Analysis cell results identifying products that need restocking based on low inventory and high sales volume." style="width:100%;" >}}
+
+{{% collapse-content title="Query breakdown" level="h4" expanded=false %}}
+This query joins inventory and sales data to identify products that need attention:
+
+- **SELECT fields**: Retrieves product IDs, store IDs, current stock levels, threshold levels, and recent sales quantities
+- **JOIN condition**: Matches inventory and sales records by product ID and store ID
+- **WHERE clause**: Filters for products where:
+  - Current stock is below the defined threshold level (indicating potential stockout)
+  - Recent sales exceed 30 units (indicating high demand)
+{{% /collapse-content %}}
+
+### 2. Analyzing inventory by geographic region
+
+Reference Tables allow you to enrich your log data with regional context. Create another Analysis cell with this query:
+
+```sql
+SELECT
+  lsa.inventory_product_id,
+  location.store_country,
+  sum(lsa.sales_quantity_sold)
+FROM
+  low_stock_analysis AS lsa
+JOIN
+  store_location_data AS location
+  ON inventory_store_id = location.store_id
+GROUP BY lsa.inventory_product_id, location.store_country
+```
+
+{{< img src="/logs/workspace/use_cases/analyze_inventory_across_regions/retail_location_analysis_cell.png" alt="SQL query joining inventory data with store location reference table to analyze sales by geographic region" style="width:100%;" >}}
+
+{{% collapse-content title="Query breakdown" level="h4" expanded=false %}}
+This query:
```

Comment:
I liked the specificity of how the previous query was explained, so I could understand what each operator was doing. I don't think it's strictly necessary to do the exact same thing here again, but for someone who needs a query breakdown, the descriptions of what the different functions do could help them tweak it for their needs!